src/test/ui/numbers-arithmetic/saturating-float-casts.rs

```diff
@@ -1,15 +1,19 @@
 // run-pass
 // Tests saturating float->int casts. See u128-as-f32.rs for the opposite direction.
-// compile-flags: -Z saturating-float-casts
+//
+// Some of these tests come from a similar file in miri,
+// tests/run-pass/float.rs. They're just duplicated currently but we may want
```

Comment:
Miri duplicates quite a few rustc tests for basic language operations, since so far we don't have a good way to run the rustc tests in Miri (Cc https://github.com/rust-lang/miri/issues/55). So this is just no exception. :)

Reply:
Or did you mean duplication within this file? If so, yeah I agree.